### PR TITLE
for bcscRedirect page

### PR DESCRIPTION
**Pull request by peggy-gov-bc-ca**
# Description

Add BCSC Redirect page

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Tested locally
image

# Checklist:

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: SPDBCSC-261

### CODE REVIEW

File: src/ecrc-frontend/src/components/base/button/Button.css [hunk 1]
```diff
@@ -29,3 +29,19 @@
...
+
+.ecrc_common_btn {
+  color: white;
+  width: 139px;
+  height: 40px;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 14px;
+  background-color: rgb(0, 51, 102);
+  padding: 0;
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/base/button/Button.css]: dont think this is required

File: src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+  const requestBtn = {
+    label: "REQUEST FORM",
+    buttonStyle: "btn ecrc_common_btn",
+    buttonSize: "btn",
+    type: "button"
+  };
+
+  const onLoginClick = () => {
+    window.open("https://support.wwf.org.uk", "_blank");
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.js]: what does `_blank` do in this function?

File: src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { create } from "react-test-renderer";
+
+import BcscRedirect from "./BcscRedirect";
+
+describe("Consent Page Component", () => {
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js]: this is not the consent page

File: src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  test("Matches the snapshot", () => {
+    const header = {
+      name: "Criminal Record Check"
+    };
+
+    const page = {
+      header
+    };
+
+    const consent = create(<BcscRedirect page={page} />);
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js]: same here, change this to match the component name being tested 😄

File: src/ecrc-frontend/src/components/page/consent/Consent.js [hunk 5]
```diff
@@ -21,7 +21,7 @@ export default function Consent({ page: { header }, onContinueClick }) {
   useEffect(() => {
     if (firstBoxChecked && secondBoxChecked && thirdBoxChecked) {
       setInputEnabled("textinput_editable_white");
-      if (applicantName) {
+      if (applicantName !== "") {
```
Comment (sdevalapurkar-bcgov) [inline src/ecrc-frontend/src/components/page/consent/Consent.js]: why was this changed? did you need to explicitly check for nulls?

File: src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.js (hunk 2 above)
Comment (peggy-gov-bc-ca) [reply src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.js]: _blank will start a new browser tab.
That is the behaviour in mock page

File: src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js (hunk 3 above)
Comment (peggy-gov-bc-ca) [reply src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js]: sry. stupid error

File: src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js (hunk 4 above)
Comment (peggy-gov-bc-ca) [reply src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js]: right

File: src/ecrc-frontend/src/components/page/consent/Consent.js (hunk 5 above)
Comment (peggy-gov-bc-ca) [reply src/ecrc-frontend/src/components/page/consent/Consent.js]: no, no, wrong branch. so, something is wrong.
will change back.

**Comment by peggy-gov-bc-ca** [comment]
the button overflow issue is fixed with use of Bootstrap css.
